packages/components/src/components/helperlabel/helperlabel.styles.ts

```diff
@@ -0,0 +1,44 @@
+import { css } from 'lit';
+import { hostFitContentStyles, hostFocusRingStyles } from '../../utils/styles';
+
+const styles = [hostFitContentStyles, css`
+
+  :host {
+    flex-direction: column;
+    align-items: flex-start;
+    gap: 0.5rem;
+  }
+
+  :host([disabled]) .mdc-label,
+  :host([disabled]) .footer {
+    color: var(--mds-color-theme-text-primary-disabled);
+  }
+
```

Comment:
This style is missing, this is default gray color of the helper text.
```suggestion
  .footer {
    color: var(--mds-color-theme-text-secondary-normal);
  }

```